### PR TITLE
sane_backends: bump to 1.0.30

### PR DESCRIPTION
**Pull request by kenmays**
Tested build.
```
### Backends

 - `epson2`: fixes CVE-2020-12867 (GHSL-2020-075) and several memory
   management issues found while addressing that CVE
 - `epsonds`: addresses out-of-bound memory access issues to fix
   CVE-2020-12862 (GHSL-2020-082) and CVE-2020-12863 (GHSL-2020-083),
   addresses a buffer overflow fixing CVE-2020-12865 (GHSL-2020-084)
   and disables network autodiscovery to mitigate CVE-2020-12866
   (GHSL-2020-079), CVE-2020-12861 (GHSL-2020-080) and CVE-2020-12864
   (GHSL-2020-081).  Note that this backend does not support network
   scanners to begin with.
 - `magicolor`: fixes a floating point exception and uninitialized data
   read
 - fixes an overflow in `sanei_tcp_read()`
```

### CODE REVIEW

File: media-gfx/sane_backends/sane_backends-1.0.30.recipe [hunk 1]
```diff
@@ -18,12 +18,12 @@ HOMEPAGE="http://www.sane-project.org"
...
 REVISION="1"
-SOURCE_URI="https://gitlab.com/sane-project/backends/uploads/54f858b20a364fc35d820df935a86478/sane-backends-$portVersion.tar.gz"
-CHECKSUM_SHA256="aa027b4e5f59849cd41b8c26d54584cf31fffd986049019be6ad4140e11ea8ed"
+SOURCE_URI="https://gitlab.com/sane-project/backends/uploads/c3dd60c9e054b5dee1e7b01a7edc98b0/sane-backends-$portVersion.tar.gz"
+CHECKSUM_SHA256="3f5d96a9c47f6124a46bb577c776bbc4896dd17b9203d8bfbc7fe8cbbcf279a3"
 SOURCE_DIR="sane-backends-$portVersion"
 PATCHES="sane_backends-$portVersion.patchset"
 
-ARCHITECTURES="?x86_gcc2 x86 x86_64"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
```
Comment (Begasus) [inline media-gfx/sane_backends/sane_backends-1.0.30.recipe:26]: Parse errors on gcc2, probably could be fixed, question is, do you want to take the time to fix them? :)

**Comment by Vidrep** [comment]
I was able to build the hpkg successfully on 64 bit. Once installed it worked as the previous revision. On 32 bit (setarch x86) the build failed.

**Comment by Vidrep** [comment]
/sources/sane-backends-1.0.30/sanei/sanei_usb.c: In function `sanei_usb_init':
/sources/sane-backends-1.0.30/sanei/sanei_usb.c:1399: warning: assignment makes pointer from integer without a cast
/sources/sane-backends-1.0.30/sanei/sanei_usb.c: In function `sanei_usb_add_endpoint':
/sources/sane-backends-1.0.30/sanei/sanei_usb.c:2222: parse error before `*'
/sources/sane-backends-1.0.30/sanei/sanei_usb.c:2229: `ep_in' undeclared (first use in this function)
/sources/sane-backends-1.0.30/sanei/sanei_usb.c:2229: (Each undeclared identifier is reported only once
/sources/sane-backends-1.0.30/sanei/sanei_usb.c:2229: for each function it appears in.)
/sources/sane-backends-1.0.30/sanei/sanei_usb.c:2230: `ep_out' undeclared (first use in this function)
/sources/sane-backends-1.0.30/sanei/sanei_usb.c:2247: `transfer_type_msg' undeclared (first use in this function)
/sources/sane-backends-1.0.30/sanei/sanei_usb.c: In function `sanei_usb_open':
/sources/sane-backends-1.0.30/sanei/sanei_usb.c:2722: warning: `transfer_type' might be used uninitialized in this function
/sources/sane-backends-1.0.30/sanei/sanei_usb.c: In function `sanei_usb_reset':
/sources/sane-backends-1.0.30/sanei/sanei_usb.c:3122: parse error before `int'
/sources/sane-backends-1.0.30/sanei/sanei_usb.c:3124: `ret' undeclared (first use in this function)
/sources/sane-backends-1.0.30/sanei/sanei_usb.c: At top level:
/sources/sane-backends-1.0.30/sanei/sanei_usb.c:2202: warning: `sanei_usb_transfer_type_desc' defined but not used
Makefile:530: recipe for target 'sanei_usb.lo' failed
make[1]: *** [sanei_usb.lo] Error 1
make[1]: Leaving directory '/sources/sane-backends-1.0.30/sanei'
Makefile:597: recipe for target 'all-recursive' failed
make: *** [all-recursive] Error 1
Warning: Command '['bash', '-c', '. /wrapper-script']' returned non-zero exit status 2.
Error: Build has failed - stopping.
[x86] ~/haikuports>

**Comment by kenmays** [comment]
@Vidrep - sane-backends-1.0.30 packages built fine on hrev54326 x86 on Jun 18th. Is this issue on R1B2 x86?

**Comment by Begasus** [comment]
> I was able to build the hpkg successfully on 64 bit. Once installed it worked as the previous revision. On 32 bit (setarch x86) the build failed.

Those are the errors I mentioned earlier for gcc2

**Pull request merged**